code/run_capsule.py

```diff
@@ -187,15 +188,15 @@ def write_input_model_list(
     if tasks_per_job < 1:
         raise ValueError("tasks_per_job must be at least 1")
 
-    num_actual_workers = math.ceil(len(input_model_list) / tasks_per_job)
-    jobs_for_each_worker = np.array_split(input_model_list, num_actual_workers)
+    number_of_jobs = math.ceil(len(input_model_list) / tasks_per_job)
+    jobs_for_each_worker = np.array_split(input_model_list, number_of_jobs)
     logger.info(f"Tasks per job: {tasks_per_job}")
 
     # Step 2: Write output per job inside worker folder
     for worker_id, job_group in enumerate(
```

Comment:
this still uses the old terminology - we really shouldn't talk about workers at all here, as nextflow handles that. this should be just tasks for each job, job folders, etc.